applications/imaging_ai_segmentator/README.md

```diff
@@ -1,198 +1,157 @@
-
-
-Set up the Holohub environment, if not already done
-```bash
-./run setup
-```
-
-Set the environment variables for the application
 ```bash
-source applications/imaging_ai_segmentator/env_settings.sh
+HOLOSCAN_INPUT_PATH=/workspace/holohub/data/imaging_ai_segmentator/dicom
+HOLOSCAN_MODEL_PATH=/workspace/holohub/data/imaging_ai_segmentator/models
+HOLOSCAN_OUTPUT_PATH=/workspace/holohub/output
 ```
 
-If not already done, download images of a CT series from [TCIA](https://nbia.cancerimagingarchive.net/nbia-search/), unzip if necessary, and save the folder of DICOM files under the folder `$HOLOSCAN_INPUT_PATH`.
+You can modify them by setting the right env variable and mount the right voluems, for instance:
 
-Optionally download the AI model from [MONAI Model Zoo](https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBody_ct_segmentation), or wait till the build step to have it downloaded automatically
-
-```bash
-mkdir -p $HOLOSCAN_MODEL_PATH
-pip install gdown
-python -m gdown https://drive.google.com/uc?id=1PHpFWboimEXmMSe2vBra6T8SaCMC2SHT -O $HOLOSCAN_MODEL_PATH/model.pt
-```
-
-Install Python packages required by the application
 ```bash
-pip install -r applications/imaging_ai_segmentator/requirements.txt
+./dev_container build_and_run imaging_ai_segmentator --container_args "-v /local/output:/my_output -e HOLOSCAN_OUTPUT_PATH=/my_output"
 ```
 
-Build and install the application
-```bash
-./dev_container build_and_install imaging_ai_segmentator
-```
+### Development Environment Setup
 
-Run the application
-```bash
-rm -f -r $HOLOSCAN_OUTPUT_PATH
-python install/imaging_ai_segmentator/app.py
-```
+You can run the application either in your local development environment or inside the Holohub development container. The steps are nearly identical, with only the first step differing.
 
-**_Note_**
-If desired, run the application with explicitly input, output, and/or model folder path, for example
-```bash
-rm -f -r ./output
-python install/imaging_ai_segmentator/app.py -m $HOLOSCAN_MODEL_PATH -i $HOLOSCAN_INPUT_PATH -o ./output
-```
+1. **Set up the Holohub environment:**
 
-Check output
-```bash
-ls $HOLOSCAN_OUTPUT_PATH
-```
+   A. **Within Container:** Build and launch the Holohub Container:
 
-There should be a DICOM segmentation file with randomized file name. There should also a `saved_images_folder` containing folder named after the input DICOM series' instance UID, which in turn contains the input and segmentation image files in NIfTI format, e.g.
-```bash
-applications/imaging_ai_segmentator/output
-├── 1.2.826.0.1.3680043.10.511.3.64271669147396658491950188504278234.dcm
-└── saved_images_folder
-    └── 1.3.6.1.4.1.14519.5.2.1.7085.2626
-        ├── 1.3.6.1.4.1.14519.5.2.1.7085.2626.nii
-        └── 1.3.6.1.4.1.14519.5.2.1.7085.2626_seg.nii
-```
+   ```bash
+   ./dev_container launch
```

Comment:
Fails, because if the user follows the README to the tee, the default dev container has not been built yet.

Reply:
@jjomier Although we can replace `./dev_container` command with corresponding `./holohub` command in most of situations, it does not have full coverage yet, so maybe a bit too early to deprecate `./dev_container`. For instance, `./holohub install` has not been implemented yet.

Reply:
@MMelQin this command is now replaced by `./holohub run-container imaging_ai_segmentator`, which pulls the latest Holoscan SDK docker image, create Holohub docker container, set up requirement for this application, and run the container.